seatunnel-connectors-v2-dist/pom.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
We don't need this file anymore. As an alternative, you need to add this connector to `seatunnel-dist/pom.xml`.